src/App.js

```diff
@@ -54,6 +55,7 @@ const App = () => {
           element={token ? <History /> : <Navigate replace to="/login" />}
         />
         <Route path="/library" element={<Library />} />
+        <Route path="/watchlater" element={<WatchLater />} />

```

Comment:
You can put this in protected route so if user is not logged in he/she will be redirected to login page

Reply:
Has been fixed